### PR TITLE
update links page

### PR DESCRIPTION
**Pull request by MattyTheHacker**

### CODE REVIEW

File: _data/links.yml [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+  url: /ball
+  image: /assets/tex/raster/tex_ball.png
+
+- text: "Grab your ticket!"
+  url: /ball/2024/tickets
+  image: /assets/tex/raster/tex_ball.png
+
+- type: divider
+  text: "General"
+
```
Comment (codemicro) [inline _data/links.yml:14]: Ball pages aren't ready nor live yet

**Review by codemicro — DISMISSED** [review]
See above

**Comment by MattyTheHacker** [comment]
ye should have clarified this was dependant on #553

**Comment by MattyTheHacker** [comment]
@codemicro re-review plox

**Review by codemicro — APPROVED** [review]

**Pull request merged**